bigbluebutton-html5/private/locales/en.json

```diff
@@ -363,18 +363,18 @@
     "app.actionsBar.currentStatusDesc": "current status {0}",
     "app.actionsBar.captions.start": "Start viewing closed captions",
     "app.actionsBar.captions.stop": "Stop viewing closed captions",
-    "app.audioNotification.audioFailedError1001": "Error 1001: WebSocket disconnected",
-    "app.audioNotification.audioFailedError1002": "Error 1002: Could not make a WebSocket connection",
-    "app.audioNotification.audioFailedError1003": "Error 1003: Browser version not supported",
-    "app.audioNotification.audioFailedError1004": "Error 1004: Failure on call (reason={0})",
-    "app.audioNotification.audioFailedError1005": "Error 1005: Call ended unexpectedly",
-    "app.audioNotification.audioFailedError1006": "Error 1006: Call timed out",
-    "app.audioNotification.audioFailedError1007": "Error 1007: ICE negotiation failed",
-    "app.audioNotification.audioFailedError1008": "Error 1008: Transfer failed",
-    "app.audioNotification.audioFailedError1009": "Error 1009: Could not fetch STUN/TURN server information",
-    "app.audioNotification.audioFailedError1010": "Error 1010: ICE negotiation timeout",
-    "app.audioNotification.audioFailedError1011": "Error 1011: ICE gathering timeout",
-    "app.audioNotification.audioFailedError1012": "Error 1012: ICE connection closed",
+    "app.audioNotification.audioFailedError1001": "WebSocket disconnected (error 1001)",
+    "app.audioNotification.audioFailedError1002": "Could not make a WebSocket connection (error 1002)",
+    "app.audioNotification.audioFailedError1003": "Browser version not supported (error 1003)",
+    "app.audioNotification.audioFailedError1004": "Failure on call (reason={0}) (error 1004)",
+    "app.audioNotification.audioFailedError1005": "Call ended unexpectedly (error 1005)",
+    "app.audioNotification.audioFailedError1006": "Call timed out (error 1006)",
+    "app.audioNotification.audioFailedError1007": "Connection negotiation failed (ICE error 1007)",
+    "app.audioNotification.audioFailedError1008": "Transfer failed (error 1008)",
+    "app.audioNotification.audioFailedError1009": "Could not fetch STUN/TURN server information (error 1009)",
+    "app.audioNotification.audioFailedError1010": "Connection negotiation timeout (ICE error 1010)",
+    "app.audioNotification.audioFailedError1011": "Connection timeout (ICE error 1011)",
```

Comment:
This one isn't really accurate, but might be fine since it doesn't really happen.